File: app/src/main/java/com/example/movies/ui/actorsdetailtabs/view/TvShowsFragment.java
package com.example.movies.ui.actorsdetailtabs.view;

import android.os.Bundle;

import com.example.movies.core.base.BaseActorsShowFragment;

public class TvShowsFragment extends BaseActorsShowFragment {

    @Override
    public void fetchActorsShows() {
        service.fetchActorsTv(id, this);
    }

    public static TvShowsFragment newInstance(int id) {
        TvShowsFragment tvShowsFragment = new TvShowsFragment();
        Bundle bundle = new Bundle();
        bundle.putInt("id", id);
        tvShowsFragment.setArguments(bundle);
        return tvShowsFragment;
    }


}

File: app/src/main/java/com/example/movies/ui/actorsdetailtabs/view/TvShowsFragment.kt
package com.example.movies.ui.actorsdetailtabs.view

import android.os.Bundle

import com.example.movies.core.base.BaseActorsShowFragment

class TvShowsFragment : BaseActorsShowFragment() {

    override fun fetchActorsShows() {
        id?.let { id -> service.fetchActorsTv(id, this) }

    }

    companion object {

        fun newInstance(id: Int): TvShowsFragment {
            val tvShowsFragment = TvShowsFragment()
            val bundle = Bundle()
            bundle.putInt("id", id)
            tvShowsFragment.arguments = bundle
            return tvShowsFragment
        }
    }


}
